clustered/client/src/test/java/org/ehcache/clustered/client/internal/service/ClusterStateRepositoryReplicationTest.java

```diff
@@ -136,7 +136,7 @@ public void testClusteredStateRepositoryReplicationWithSerializableKV() throws E
     ClusteringService.ClusteredCacheIdentifier spaceIdentifier = (ClusteringService.ClusteredCacheIdentifier) service.getPersistenceSpaceIdentifier("test",
       config);
 
-    ServerStoreProxy serverStoreProxy = service.getServerStoreProxy(spaceIdentifier, new StoreConfigurationImpl<>(config, 1, null, null), Consistency.STRONG);
+    ServerStoreProxy serverStoreProxy = service.getServerStoreProxy(spaceIdentifier, new StoreConfigurationImpl<>(config, 1, null, null), Consistency.STRONG, null);
```

Comment:
To prevent the null passing, a NopCallback will be nice and a default method
```
  default <K, V> ServerStoreProxy getServerStoreProxy(ClusteredCacheIdentifier cacheIdentifier, final Store.Configuration<K, V> storeConfig, Consistency consistency) throws CachePersistenceException {
    return getServerStoreProxy(ClusteredCacheIdentifier cacheIdentifier, final Store.Configuration<K, V> storeConfig, Consistency consistency, NOP_CALLBACK);
  }
```

Reply:
Would it ever make sense to not provide a callback though?  Even if you had no caching layer to invalidate clients still need to respond to the resolve requests.

Reply:
I'm not that into it. What else are they doing with it? But then, null shouldn't be allowed (and nop could be used for testing)